Composer/packages/extensions/adaptive-flow/src/adaptive-flow-editor/renderers/NodeWrapper.tsx

```diff
@@ -75,7 +78,16 @@ export const ActionNodeWrapper: FC<NodeWrapperProps> = ({ id, tab, data, onEvent
       aria-label={generateSDKTitle(data, '', tab)}
       onClick={(e) => {
         e.stopPropagation();
-        onEvent(NodeEventTypes.Focus, { id, tab });
+        e.preventDefault();
+
+        const payload = { id, tab };
+        if (e.ctrlKey) {
```

Comment:
Do we want to also check for `e.metaKey` so Mac users can use Cmd+Click?

Reply:
resolved